src/riscv/lib/src/jit.rs

```diff
@@ -376,6 +378,8 @@ mod tests {
         xregisters: Vec<(NonZeroXRegister, u64)>,
         #[debug(skip)]
         assert_hook: Option<Box<AssertHook>>,
+        #[debug(skip)]
+        expected_jit_counters: JitTestCounters,
```

Comment:
This supports `Debug` if I'm not mistaken. It doesn't need to be skipped, right?